src/display/text_layer.js

```diff
@@ -456,6 +469,25 @@ class TextLayer {
     return canvasContext;
   }
 
+  /**
+   * @returns {number} The minimum font size enforced by the browser
+   */
+  static #ensureMinFontSizeComputed() {
+    if (this.#minFontSize === null) {
+      const div = document.createElement("div");
+      div.style.opacity = 0;
+      div.style.lineHeight = 1;
+      div.style.fontSize = "1px";
+      div.textContent = "X";
+      document.body.append(div);
+      // In `display:block` elements contain a single line of text,
+      // the height matches the line height (which, when set to 1,
+      // matches the actual font size).
+      this.#minFontSize = div.getBoundingClientRect().height;
+      div.remove();
+    }
```

Comment:
Maybe use an early return here?
```suggestion
    if (this.#minFontSize !== null) {
      return;
    }
    const div = document.createElement("div");
    div.style.opacity = 0;
    div.style.lineHeight = 1;
    div.style.fontSize = "1px";
    div.textContent = "X";
    document.body.append(div);
    // In `display:block` elements contain a single line of text,
    // the height matches the line height (which, when set to 1,
    // matches the actual font size).
    this.#minFontSize = div.getBoundingClientRect().height;
    div.remove();
```